src/traces/bar/calc.js

```diff
@@ -19,12 +19,14 @@ module.exports = function calc(gd, trace) {
     var ya = Axes.getFromId(gd, trace.yaxis || 'y');
     var size, pos;
 
+    var msUTC = !!(trace.base || trace.base === 0);
+
     if(trace.orientation === 'h') {
-        size = xa.makeCalcdata(trace, 'x');
-        pos = ya.makeCalcdata(trace, 'y');
+        size = xa.makeCalcdata(trace, 'x', msUTC);
+        pos = ya.makeCalcdata(trace, 'y', msUTC);
     } else {
-        size = ya.makeCalcdata(trace, 'y');
-        pos = xa.makeCalcdata(trace, 'x');
+        size = ya.makeCalcdata(trace, 'y', msUTC);
+        pos = xa.makeCalcdata(trace, 'x', msUTC);
```

Comment:
`pos` should not get `msUTC`, only `size`. This would matter only in a very weird situation: The position axis has type date and data as milliseconds; this should be treated as local milliseconds whether or not the data has a `base` applied along the size axis (which may have any type, date or numeric).

Reply:
Oops! Fixed in https://github.com/plotly/plotly.js/pull/4900/commits/2ae0fdfb53dfd3ce600347a2924f25793c4a7b5d.